magento/adobe-stock-integration
# #398: "See more" link applies Series filter to the grid

### Description (*)

This PR will fix the issue or enhance following functionality
1) Loader added when series ajax request initiated
2) See more link added in the series image list and initiate filter based on current serie_id
3) Fixed issue with a view all keyword as the user is not able to view all the keyword when user click on view all button in the last image in the grid

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#398: "See more" link applies Series filter to the grid

### Manual testing scenarios (*)

1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Click on filter button
5. Series id filter is not visible
6. Click on any image
7. User sees the see more link
8. On clicking "see more", serie_id filter is initiated
